### DISCUSSION TITLE
Warning: Demand 'ODV 46': Failing coordination

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#677] [Q&A]
Hi,
after generating the supply plan I noticed that some requests were not fully planned (only 5 pieces out of 15 total). I tried to check the log file but I don't understand the cause. The log report says "Failing coordination" after some planning iterations. Could you help me understand the problem? I attach the log file
[frepple-20241106111325.txt](https://github.com/user-attachments/files/17645693/frepple-20241106111325.txt)

**Comment by jdetaeye** [comment]
This error comes from deep inside the planning algorithm, and you can't easily fix it directly.
Settings that can avoid this situation:
- set the "miniumum shipment" on the sales orders to a big enough quantity (or even the full requested quantity) to avoid small partial deliveries.
- assure the "max_early" window is sufficiently high on your resources.
